### ISSUE TITLE
LDAP Sync to import LDAP groups for account permissions.

### ISSUE THREAD

**Issue by rcmcdonald91** [grokability/snipe-it#7786]
**Server (please complete the following information):**
 - Snipe-IT Version: v4.8.0 - build 4186 (master)
 - OS: Ubuntu 18.04
 - Web Server: Apache
 - PHP Version: 7.2.24-0ubuntu0.18.04.1

LDAP Sync should include a way to scope a particular OU for LDAP groups to be imported.

SnipeIT 'account permission groups' should, in term, be associated with these LDAP Groups to automatically apply role permissions to users based on their LDAP group membership.

**Comment by gilesytheking** [comment]
Yes I agree. My ldap users are assigned into groups which allow the setting of access rights etc. 

Would be great to be able to map my LDAP groups to a Snipe IT group upon sync. Currently I have to add each imported employee to a snipe IT group manually after LDAP sync.

**Comment by viclou** [comment]
+1 FD15111

**Comment by WarriorXK** [comment]
I'd love to have this feature, would make my life a lot easier.

**Comment by niklasskoldmark** [comment]
This is very important, it makes no sense and is prone to error to try to maintain s separate groups hierarchy in Snipe-IT when we have groups for the roles in AD

**Comment by rcmcdonald91** [comment]
Yes

**Comment by niklasskoldmark** [comment]
yes

**Comment by WarriorXK** [comment]
yes
